sqs/sqs_test.go

```diff
@@ -0,0 +1,113 @@
+package sqs
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/sqs"
+	"github.com/italolelis/outboxer"
+)
+
+func TestSQS_EventStream(t *testing.T) {
```

Comment:
We could potentially start using [table tests](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests). Wdyt?

Reply:
that's perfect! Adjusted!